src/openvic-simulation/military/Unit.cpp

```diff
@@ -203,41 +203,41 @@ bool UnitManager::generate_modifiers(ModifierManager& modifier_manager) const {
 	bool ret = true;
 
 	const auto generate_stat_modifiers = [&modifier_manager, &ret](std::string_view identifier, Unit::type_t type) -> void {
-
-		const auto stat_modifier = [&modifier_manager, &ret, &identifier](std::string_view suffix, bool positive_good,
-			ModifierEffect::format_t format) -> void {
+		const auto stat_modifier = [&modifier_manager, &ret, &identifier](
+									   std::string_view suffix, bool is_positive_good, ModifierEffect::format_t format
+								   ) -> void {
```

Comment:
```suggestion
		const auto stat_modifier = [&modifier_manager, &ret, &identifier](
			std::string_view suffix, bool is_positive_good, ModifierEffect::format_t format
		) -> void {
```
Can you remove the extra indentation here, it's an annoying thing the formatter does that we couldn't find a way to turn off. In general indentation should only increase by 1 tab character compared to the previous line.

Reply:
I couldn't find the setting either. I'll apply your comment. I think vs code will just place it like that again the next time someone touches this.